### DISCUSSION TITLE
django ratelimit backend support

### DISCUSSION THREAD

**Discussion by asesh** [django-recaptcha/django-recaptcha#257] [Q&A]
How do I make django recaptcha support django ratelimit backend so that they coordinate with each other? @morgs @danielgatis @denz @peterp @davepeck  

Thanks

**Comment by 9mido** [comment]
I too would like to know this. 

Does it also work with Cloudflare since everyone using the proxy is behind the same IP address? It might be a problem in Cloudflare because everyone is behind one IP address for the rate limiting.
